ebu_tt_live/documents/test/deduplicator-blackbox.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Deduplicator removes duplicated style and region elements
+
+  Examples:
+  | xml_file      |
+  | test_file.xml |
+
+  #Everything goes right
+  Scenario: The deduplicator successfully removes instances of element duplication from a file
```

Comment:
there does not seem to be variables in your scenario that matches up with the examples below it?